test-cases/upgrades/rolling-upgrade.yaml

```diff
@@ -3,12 +3,12 @@
 test_duration: 180
 
 # workloads
-write_stress_during_entire_test: cassandra-stress write no-warmup cl=QUORUM n=60000000 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..60000000 -log interval=5
-verify_stress_after_cluster_upgrade: cassandra-stress read no-warmup cl=QUORUM n=60000000 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..60000000 -log interval=5
+write_stress_during_entire_test: cassandra-stress write no-warmup cl=QUORUM n=100000000 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..100000000 -log interval=5
+verify_stress_after_cluster_upgrade: cassandra-stress read no-warmup cl=QUORUM n=100000000 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..100000000 -log interval=5
 prepare_write_stress: cassandra-stress write no-warmup cl=QUORUM n=10000000 -schema 'replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000 -log interval=5
 stress_cmd_read_cl_quorum: cassandra-stress read no-warmup cl=QUORUM n=10000000 -schema 'replication(factor=3) compression=DeflateCompressor' -port jmx=6868 -mode cql3 native compression=none -rate threads=1000 -pop seq=1..10000000 -log interval=5
 stress_cmd_read_10m: cassandra-stress read no-warmup cl=QUORUM duration=10m -schema 'replication(factor=3) compression=SnappyCompressor' -port jmx=6868 -mode cql3 native compression=snappy -rate threads=1000 -pop seq=1..10000000 -log interval=5
-stress_cmd_read_20m: cassandra-stress read no-warmup cl=QUORUM duration=20m -schema 'replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000 -log interval=5
+stress_cmd_read_80m: cassandra-stress read no-warmup cl=QUORUM duration=80m -schema 'replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000 -log interval=5
```

Comment:
Why?

Reply:
I explained in the commitlog: https://github.com/scylladb/scylla-cluster-tests/pull/1438/commits/0c89a8db8a4cc6804f6a96f0954e2af31d897de6